Kajabi/sage-lib
# [SAGE-1136] Docs Homepage Image Optimizations

## Description
Image assets on the docs homepage can be further compressed and properly sized to reduce payload size and improve the clarity of images. 

The adjustments will give us a reduction of `~83kb` off the initial download size.


## Screenshots
The updates will cause no major visual changes (properly sized images will be a bit more clear).


## Testing in `sage-lib`

- Navigate to http://localhost:4000/pages/index (HOMEPAGE)
- Verify hero background image, sage icon, and sage UI images load as expected.

Note: A quick way to check the clarity as well is to open localhost in one tab and the live site in another tab. Switch back and forth between tabs.


## Testing in `kajabi-products`
N/A
1. (**LOW**) This is a docs only update and should not affect the kajabi-product repo in any way.

## Related
#1136
